app/utils/rxjs/RxjsPeer.client.ts

```diff
@@ -236,13 +240,19 @@ export class RxjsPeer {
 				pushedTrackPromise = this.pushTrackDispatcher
 					.doBulkRequest({ trackName, transceiver }, (tracks) =>
 						this.taskScheduler.schedule(async () => {
-							await peerConnection.setLocalDescription(
-								await peerConnection.createOffer()
+							// create an offer
+							const offer = await peerConnection.createOffer()
+							// Turn on Opus DTX to save bandwidth
+							offer.sdp = offer.sdp!.replace(
```

Comment:
```suggestion
							offer.sdp = offer.sdp?.replace(
```

Same question here — should we use optional chaining here instead?

Reply:
Fixed in subsequent push.